config/build/rollup.config.js

```diff
@@ -179,11 +176,6 @@ const createConfig = (
             autoExternal({
                 packagePath: makePackageBasedPath(name, "./package.json"),
             }),
-            // TODO(FEI-4557): Figure out how to make this plugin work so that
-            // @khanacademy/perseus-editor works in webapp.  If we enable this
-            // plugin right now, the content editor pages in webapp will fail
-            // with the following error: `TypeError: Object(...) is not a function`
-            // terser(),
```

Comment:
wut

Reply:
This was never set up properly when we moved Perseus back to this repo.  Thankfully, today with SWC the minification can be re-enabled and I was able to verify in the editor in webapp that there were no `TypeError`s. 

<img width="1541" alt="image" src="https://github.com/user-attachments/assets/ffbda117-3f2d-4f01-8b04-cc6e890bb938" />